node/tessel-export.js

```diff
@@ -526,32 +544,32 @@ Tessel.Port.prototype.uncork = function() {
   this.sock.uncork();
 };
 
-Tessel.Port.prototype.sync = function(cb) {
-  if (cb) {
+Tessel.Port.prototype.sync = function(callback) {
+  if (callback) {
     this.sock.write(new Buffer([CMD.ECHO, 1, 0x88]));
     this.enqueue({
       size: 1,
-      callback: cb
+      callback: callback
```

Comment:
This change was made in preparation for the modernizing patch set (where `{ callback: callback }` will be `{ callback }`